lib/screenplay/suppressed_predictions/suppressed_prediction.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Screenplay.SuppressedPredictions.SuppressedPrediction do
+  @moduledoc """
+  Represents a Suppressed Prediction for a given location
+  """
+  alias Screenplay.Places
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @derive {Jason.Encoder, except: [:__meta__]}
+
+  @type t() :: %__MODULE__{
+          location_id: String.t(),
+          direction_id: integer(),
+          clear_at_end_of_day: boolean()
+        }
+
+  schema "suppressed_predictions" do
+    field(:location_id, :string)
+    field(:direction_id, :integer)
+    field(:clear_at_end_of_day, :boolean)
+
+    timestamps(type: :utc_datetime)
+  end
+
+  def changeset(suppressed_prediction, attrs \\ %{}) do
+    suppressed_prediction
+    |> cast(attrs, [
+      :location_id,
+      :direction_id,
+      :clear_at_end_of_day
+    ])
+    |> validate_required([
+      :location_id,
+      :direction_id,
+      :clear_at_end_of_day
+    ])
+    |> validate_number(:direction_id,
+      less_than_or_equal_to: 2,
+      greater_than_or_equal_to: 0,
+      message: "Location Direction ID must be 0, 1 or 2 for both directions"
+    )
+    |> validate_location()
+  end
+
+  defp validate_location(changeset) do
+    location_id = get_field(changeset, :location_id)
+
+    place_ids =
```

Comment:
Just double checking here - is `location_id` and `place.id` supposed to match here?

Reply:
For JFK is this the Ashmont/Braintree split within Places? 

I'll add northbound/southbound extras if that's the case.

Reply:
Also, in terms of filtering down the places, is there a good way of doing that? Should it be filtered via some param based on Bus/Subway or something along those lines?